src/index.js

```diff
@@ -0,0 +1,112 @@
+import './style.css';
+import {
+  form, todoInput, todosListEl,
+} from './modules/variables.js';
+import showNotification from './modules/notification.js';
+import setTime from './modules/dates.js';
+
+setTime();
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+let EditTodoId = -1;
+
+/// define renderTodos function
+function renderTodos() {
+  if (todos.length === 0) {
+    todosListEl.innerHTML = '<br><center>Nothing to do!</center>';
+    showNotification('Nothing to Do');
+    return;
+  }
+  todosListEl.innerHTML = '';
+  todos.forEach((todo, index) => {
+    todosListEl.innerHTML += `
+    <div class="todo" id=${index}>
+    <i 
+      class="bi ${todo.checked ? 'bi-check-circle-fill' : 'bi-circle'}"
+      style ="color : ${todo.color} "
+      data-action='check';
+      ></i>
+    <p class="${todo.checked ? 'checked' : ''} " style ="background-color : ${todo.color}" data-action='check' >${todo.value}</p>
+    <i class="bi bi-pencil-square" data-action="edit"></i>
+    <i class="bi bi-trash" data-action="delete"></i>
+  </div>`;
```

Comment:
[Optional] I think it would be better to use ES6 classes and methods to create DOM nodes and ["appendchild"](https://www.w3schools.com/jsref/met_node_appendchild.asp) because this will help you scale your application easier as its part of [KISS](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec) principle 😊

Reply:
[Issue resolved.](https://github.com/Lucash2022/To-Do-list-review/pull/1/commits/781b03ea9c0bfe433be0380d407f67a066d9a8bf) Thank you